compiler/testData/codegen/box/ranges/stepped/unsigned/expression/downTo/emptyProgression.kt

```diff
@@ -1,6 +1,4 @@
 // Auto-generated by GenerateSteppedRangesCodegenTestData. Do not edit!
```

Comment:
This test file is auto-generated. Please, update GenerateSteppedRangesCodegenTestData and  GenerateRangesCodegenTestData (I don't see correspodning changes in this PR; sorry if I overlooked something).

Reply:
You can (probably) use partial revert of this commit: https://github.com/JetBrains/kotlin/commit/e625bb375f6a3a9d10e9d1aa23355d903c950849#diff-7642cac8c3abb165ed1add8bc6787560R73

Reply:
Thanks for the pointers! Truth is, I didn't look at the tests, I just unmuted them with `AUTOMATICALLY_UNMUTE_PASSED_TESTS` in `KotlinTestUtils`. It's fixed now.